src/Dockerfile

```diff
@@ -1,14 +1,16 @@
 # syntax=docker/dockerfile:1
 FROM python:3.7-alpine
+# may want to swap out this for a different image? causes problems with matplotlib
```

Comment:
to be nitpicky, can you add some line breaks so it's clear which comments go with which lines? or put the comments in the same line